### ISSUE TITLE
create a structured tag in an area to indicate that routes have names on the wall

### ISSUE THREAD

**Issue by scd** [theCrag/website#2014]
This would be useful information for the end user to know and also useful in prioritising topo development as these areas are less critical for topos.

**Comment by nicHoch** [comment]
Also on route level because in some areas one some routes are marked on the
rock.
Am 19.08.2015 12:32 schrieb "Simon Dale" <notifications@github.com>:

**Comment by brendanheywood** [comment]
@nicHoch what is your thoughts on this:

Yet another 'short' name version type, physical marked name #569

To me this is better because in almost every case where I've seen rock markings, it's not the name but an initial, or funky abbreviation or whatever. Knowing that names are marked is one thing, but knowing what the markings actually are seems even better. I've also seen guidebooks that have this marking in the margin, and only every couple of routes have a marking, so once you've found a couple markings at the cliff  its really easy to scan down the margin of the guide to match the right route and find where you are.

**Comment by scd** [comment]
I think we will need to do both. If the route has the same name marked on the rocks we need to be able to say that. You cannot have the same alternate name as the primary name.

**Comment by nicHoch** [comment]
Yes especially on alpine routes where the next route is some 100m away it
is very helpful to know if you have to look out for a route name or any
sign on the start.

**Comment by georg-d** [comment]
For above mentioned purposes IMHO no tag is required as some short note in the crag/sector/route description is sufficient. Do you see a practical use case that _only_ works well with tags?

If we do decide for tagging: There are many variations, i.e. combinations of at least these things I came across in central & south of Europe:
- full names, sometimes quite basic like at https://www.thecrag.com/photo/3497390631 (rectangle with green background & black text at bottom center), sometimes nicer, sometimes even with cute visualisations like https://www.thecrag.com/photo/3215002860 - both DE, Black Forest
- abbreviated names
- route IDs / numbers like "R6", e.g. in below photo 
- arrows & lines for route width definition and/or for disambiguation of neighboring routes like in https://www.thecrag.com/photo/3498250680 (red marks in the center, DE, Frankenjura)
- with and without "goodies" like grades, length,...
- information boards with topos or maps

Representing all of these variations as one tag of it's own would IMHO be less useful than having just one tag for _"some kind of info in place that help to identify routes"_ - would you really like to select a dozen tags?
